crates/fdev/src/network_metrics_server.rs

```diff
@@ -96,51 +94,54 @@ async fn push_interface(ws: WebSocket, state: Arc<ServerState>) -> anyhow::Resul
                     }
                     _ => continue,
                 };
-                match PeerChange::try_decode_fbs(&msg) {
-                    Ok(PeerChange::Error(err)) => {
-                        tracing::error!(error = %err.message(), "Received error from peer");
-                        break;
-                    }
+
+                let mut decoding_errors = String::new(); // TODO: change this to Vec<String>
```

Comment:
is this todo necessary or we can merge without it?

Reply:
I guess is better having that todo fixed. I'll do it.